clap_complete/tests/testsuite/dynamic.rs

```diff
@@ -132,6 +132,118 @@ goodbye-world"
     );
 }
 
+#[test]
+fn suggest_argument_value() {
+    let mut cmd = Command::new("dynamic")
+    .arg(
+        clap::Arg::new("input")
+            .long("input")
+            .short('i')
+            .value_hint(clap::ValueHint::FilePath),
+    )
+    .arg(
+        clap::Arg::new("format")
+            .long("format")
+            .short('F')
+            .value_parser(["json", "yaml", "toml"]),
+    )
+    .args_conflicts_with_subcommands(true);
+    
+    {
+        use std::fs;
```

Comment:
I'm generally not a fan of `use` in cases like this

Reply:
Hahaha, I don't really have a preference in coding style for this, I'll make the change.